docs/src/pages/quasar-cli-vite/state-management-with-pinia.md

```diff
@@ -148,3 +148,24 @@ export default {
 ```
 
 [More info on defining a Pinia store](https://pinia.vuejs.org/core-concepts/).
+
+## Accessing router in Pinia store
+Simply use `this.router` in Pinia stores to get access to the router.
```

Comment:
Let's leave an empty line after the title for readability(_codebase-wise_) and to match with the other usages.

Let's update the title slightly.

```suggestion
## Accessing the router in Pinia stores

Simply use `this.router` in Pinia stores to get access to the router.
```